tests/mpi/project_bv_div_conf.cc

```diff
@@ -0,0 +1,290 @@
+// ---------------------------------------------------------------------
+//                                                                      
+// Copyright (C) 2004 - 2018 by the deal.II authors                     
+//                                                                      
+// This file is part of the deal.II library.                            
+//                                                                      
+// The deal.II library is free software; you can use it, redistribute   
+// it, and/or modify it under the terms of the GNU Lesser General       
+// Public License as published by the Free Software Foundation; either  
+// version 2.1 of the License, or (at your option) any later version.   
+// The full text of the license can be found in the file LICENSE.md at  
+// the top level directory of deal.II.                                  
+//                                                                      
+// ---------------------------------------------------------------------
+
+
+// Check that project_boundary_values_div_conforming works for a 
+// distributed triangulation
+
+#include <deal.II/base/function.h>
+#include <deal.II/base/utilities.h> 
+#include <deal.II/base/conditional_ostream.h>
+
+#include <deal.II/lac/affine_constraints.h>
+
+#include <deal.II/grid/tria.h>
+#include <deal.II/grid/grid_generator.h>
+
+#include <deal.II/dofs/dof_handler.h>
+#include <deal.II/dofs/dof_renumbering.h>
+#include <deal.II/dofs/dof_tools.h>
+
+#include <deal.II/fe/fe_dgq.h>
+#include <deal.II/fe/fe_system.h>
+#include <deal.II/fe/fe_raviart_thomas.h>
+
+#include <deal.II/numerics/vector_tools.h>
+
+#include <deal.II/distributed/tria.h>
+#include <deal.II/distributed/grid_refinement.h>
+
+#include <iostream>
+
```

Comment:
You need to include the test [master header](https://github.com/dealii/dealii/blob/master/tests/mpi/attach_data_02.cc#L32) here:
```c++
#include "../tests.h"
```